ko-KR.json

```diff
@@ -125,13 +125,13 @@
 	"ADD_TO_LIBRARY_TO_GET_NOTIFIED": "라이브러리에 추가하여 사용 가능한 경우 알림을 받습니다.",
 	"GET_NOTIFIED_WHEN_AVAILABLE": "사용 가능한 경우 알림이 발송됩니다.",
 	"WARNING_STREAMING_SERVER": "경고: 로컬 스트리밍 서버를 찾을 수 없습니다. 로컬 파일, 유튜브, 비트토렌트의 스트리밍을 사용할 수 없습니다",
-	"WARNING_STREAMING_SERVER_DISMISS": "Dismiss",
-	"WARNING_STREAMING_SERVER_LATER": "Later",
+	"WARNING_STREAMING_SERVER_DISMISS": "닫기",
+	"WARNING_STREAMING_SERVER_LATER": "나중에",
 	"DECODER_WARNING": "경고: 이 비디오는 하드웨어에서 지원되지 않을 수 있습니다. 재생이 느릴 수 있습니다.",
 	"GENERIC_ERROR_MESSAGE": "Something went wrong!",
-	"TRY_AGAIN": "Try again",
-	"CLEAR_DATA": "Clear data",
-	"PLAYER_ERROR": "앗, 플레이어 오류",
+	"TRY_AGAIN": "다시 시도",
+	"CLEAR_DATA": "데이터 지우기",
+	"PLAYER_ERROR": "!!!, 플레이어 오류",
```

Comment:
this can't be right, can it?

Reply:
> this can't be right, can it?

"PLAYER_ERROR": "앗, 플레이어 오류"

In Korean, especially when conveying information, people don't seem to use onomatopoeia much. So I replaced it with exclamation marks, but to make it more general,  just 'Player Error' seems more appropriate.
"플레이어 오류" would be that.

Also, if we want to be more internet-friendly, just writing 'oops' as it sounds in Korean might be a good idea.
For this instance, it is "웁스! 플레이어 오류". However, while some people might find it funny, others might think the translation was done carelessly.